src/EventListener/InsertTagsListener.php

```diff
@@ -48,10 +48,13 @@ public function __construct(ContaoFrameworkInterface $framework)
      * Replaces calendar insert tags.
      *
      * @param string $tag
+     * @param bool   $useCache
+     * @param mixed  $cacheValue
+     * @param array  $flags
      *
      * @return string|false
      */
-    public function onReplaceInsertTags(string $tag)
+    public function onReplaceInsertTags(string $tag, bool $useCache, $cacheValue, array $flags)
```

Comment:
Again, `$useCache` and `$cacheValue` are not used anywhere and `$flags` should be optional.